maskrcnn_benchmark/structures/boxlist_ops.py

```diff
@@ -108,7 +108,12 @@ def cat_boxlist(bboxes):
         bboxes (list[BoxList])
     """
     assert isinstance(bboxes, (list, tuple))
-    assert all(isinstance(bbox, BoxList) for bbox in bboxes)
+
+    # bbox could be empty(list).
+    assert all(isinstance(bbox, (list, BoxList)) for bbox in bboxes)
```

Comment:
I'm not sure we want to accept lists as well.
We can have an empty `BoxList` now, as we have support for tensors with zero in its dimensions.

Reply:
fixed. Use empty BoxList instead of [] in retinanet_infer.py